test/parallel/test-fs-readfile-error.js

```diff
@@ -3,6 +3,11 @@ var assert = require('assert');
 var exec = require('child_process').exec;
 var path = require('path');
 
+if (process.platform === 'freebsd') {
```

Comment:
Maybe add a comment explaining why it doesn't work on FreeBSD.  (Because you can open and read from a directory, right?)  It's probably not restricted to just FreeBSD.